modules/survey_accounts/test/TestPlan.md

```diff
@@ -14,26 +14,28 @@
     Make sure that if you don’t fill out a field that is required,
     and you hit save and continue fields you have filled out are not erased.
     [Manual Testing]
-6.  Add Survey button:
+6.  Add Survey button: 
     a. Try sending the URL to yourself -  make sure that both email addresses match in order for the “Email Survey” button to work
-      (try mismatched email addresses to make sure that the "Email Survey" button remains inactive).
+      (try mismatched email addresses to make sure that the "Email Survey" button remains inactive).  
     b. Once you hit “Email Survey” you should get a blank page where you can customize an email to go along with the URL –
     enter in a message and make sure it sends (also try the cancel button on this page).
-    The email can be pre-populated for each instrument using participant_emails table.
-    c. Ensure that the instrument being sent is what you get (and that the URL brings you to the correct survey).
-    d. Use the `Create Survey` button (no email address should be specified).
-    e. Use the `Email survey` button by specifying email address.
-    f. Try mismatched PSCID and DCCID and should get an error message.
-    g. Try creating duplicate instrument for a candidate for a visit, should get an error message.
-    h. Try creating instrument for a candidate in a non-existing visit, should get an error message.
-    [Manual Testing]
+    The email can be pre-populated for each instrument using participant_emails table.  
+    c. Ensure that the instrument being sent is what you get (and that the URL brings you to the correct survey).  
+    d. Use the `Create Survey` button (no email address should be specified).  
+    e. Use the `Email survey` button by specifying email address.  
+    f. Check that the survey list shows the newly created survey with all the correct information.    
+    g. Try creating a survey for a candidate in a Project that the user is not affiliated with. This should not be allowed.  
+    h. Try mismatched PSCID and DCCID and should get an error message.  
+    i. Try creating duplicate instrument for a candidate for a visit, should get an error message.  
+    j. Try creating instrument for a candidate in a non-existing visit, should get an error message.
+    [Manual Testing]  
 7.  Enable a new instrument for Survey module and verify survey can be created. (IsDirectEntry set to 1 in test_names table).
     [Manual Testing]
-8.  Verify Status column in the module updates appropriately
+8.  Verify Status column in the module updates appropriately  
     a. When survey is created Status should be set to `Created` when no email address is specified
```

Comment:
1. When survey is created Status should be set to `Created` when no email address is specified